harness/determined/deploy/local/cluster_utils.py

```diff
@@ -259,13 +289,27 @@ def cluster_down(cluster_name: str, delete_db: bool) -> None:
 
 
 def logs(cluster_name: str, no_follow: bool) -> None:
```

Comment:
feels like this should be `follow` not `no_follow`, but that's a nit and you may ignore it if you disagree or don't care enough.